cmp/form/FormModel.ts

```diff
@@ -22,13 +30,21 @@ export interface FormConfig {
     /** Map of initial values for fields in this model. */
     initialValues?: PlainObject;
 
+    /** Options governing persistence of the form state. */
+    persistWith?: FormPersistOptions;
+
     disabled?: boolean;
     readonly?: boolean;
 
     /** @internal */
     xhImpl?: boolean;
 }
 
+export interface FormPersistOptions extends PersistOptions {
+    /** If persisting only a subset of all fields, provide an array of field names. */
+    fieldsToPersist?: string[];
```

Comment:
could consider using two properties -- fieldsToInclude and fieldsToExclude.  We have that pattern elsewhere